src/FixtureBuilder/ExpressionLanguage/Parser/TokenParser/Chainable/VariableTokenParser.php

```diff
@@ -45,7 +45,9 @@ public function canParse(Token $token): bool
      */
     public function parse(Token $token)
     {
-        $variable = substr($token->getValue(), 1);
+        $variable = empty($token->getValue())
```

Comment:
likewise here; also general note `empty()` should be avoided at all cost because is unreliable: `empty('0') = true`